packages/grid_client/src/primitives/nodes.ts

```diff
@@ -24,6 +24,7 @@ interface PublicIps {
   id: string;
   ip: string;
   contractId: number;
```

Comment:
please remove `contractId` if the `contract_id` is the one used to not be confusing

Reply:
Is it not used anywhere else?